src/credentials/FabricTable.h

```diff
@@ -561,6 +562,17 @@ class DLL_EXPORT FabricTable
      */
     CHIP_ERROR FetchRootPubkey(FabricIndex fabricIndex, Crypto::P256PublicKey & outPublicKey) const;
 
+    /**
+     * @brief Get the CASE Authenticated Tags from the NOC for the given `fabricIndex`.
+     *
+     * @param fabricIndex - Fabric for which to get the root public key (subject public key of RCAC)
+     * @param outCats - CATValues struct to write the NOC CATs for the given fabric index
+     * @retval CHIP_NO_ERROR on success
+     * @retval CHIP_ERROR_INVALID_FABRIC_INDEX if not found/available, or `fabricIndex` has a bad value
+     * @retval other CHIP_ERROR values on other invalid arguments or internal errors.
+     */
+    CHIP_ERROR FetchCats(const FabricIndex fabricIndex, CATValues & outCats) const;
```

Comment:
Nit: While I support fetching Cats, shouldn't these CATs be upper case?

Reply:
That is the prevailing pattern in the sdk.  I'll update.

Reply:
Updated to match prevailing style, which is:

* Method names, object types: CATs
* variables, arguments: cats